### ISSUE TITLE
Error: 	uses-sdk:minSdkVersion 16 cannot be smaller than version 21 declared in library [com.facebook.react:react-android:0.73.0]

### ISSUE THREAD

**Issue by pratikdonda** [dev-family/react-native-device-country#11]
Getting Error:
	uses-sdk:minSdkVersion 16 cannot be smaller than version 21 declared in library [com.facebook.react:react-android:0.73.0]

using :
"react-native": "0.73.0",

minSdkVersion = 24
compileSdkVersion = 33
targetSdkVersion = 33
kotlinVersion = "1.8.0"

android studio : 2024.1.2 koala

**Comment by devthejo** [comment]
As a workaround I use `yarn patch` (yarn berry)

`package.json`
```json
"resolutions": {
    "react-native-device-country@^1.0.4": "patch:react-native-device-country@npm%3A1.0.4#./.yarn/patches/react-native-device-country-npm-1.0.4-53c0d4f775.patch"
  },
```

`.yarn/patches/react-native-device-country-npm-1.0.4-53c0d4f775.patch`
```diff
diff --git a/android/build.gradle b/android/build.gradle
index 687e5ff8c2769527f4cb13a48c1efd8d9328ded6..203eeea288844ba2277a318935550b588f2b26bf 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -18,11 +18,11 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('DeviceCountry_compileSdkVersion', 31)
+    compileSdkVersion safeExtGet('DeviceCountry_compileSdkVersion', 33)
     buildToolsVersion safeExtGet('DeviceCountry_buildToolsVersion', '29.0.2')
     defaultConfig {
-        minSdkVersion safeExtGet('DeviceCountry_minSdkVersion', 16)
-        targetSdkVersion safeExtGet('DeviceCountry_targetSdkVersion', 31)
+        minSdkVersion safeExtGet('DeviceCountry_minSdkVersion', 23)
+        targetSdkVersion safeExtGet('DeviceCountry_targetSdkVersion', 33)
         versionCode 1
         versionName "1.0"
```

don't forget to run `yarn install` ;-)

**Comment by mshekspirtt** [comment]
This fix is included in version 1.0.5. Please update to the latest version and let us know if you encounter any further issues.
